### ISSUE TITLE
Support .skipIf().concurrent

### ISSUE THREAD

**Issue by beorn** [vitest-dev/vscode#36]
**Describe the bug**

Tests that use Vitest's [.skipIf](https://vitest.dev/api/#test-skipif) (and I presume [.runIf](https://vitest.dev/api/#test-runif)) show up failed because "Test result not found.". 

**To Reproduce**

Try for example:

```ts
const isDev = true

describe.skipIf(isDev)("suite 1", () => {
  test("skipped if isDev", () => {
    expect(true).toBe(true)
  })
})

describe("suite 2", () => {
  test.skipIf(isDev)("also skipped if isDev", () => {
    expect(true).toBe(true)
  })
})
```

**Expected behavior**

Tests like this should not automatically err out.

Ideally they should be ignored or included depending on the result of the condition (perhaps assume that a missing test result means that it was skipped?) - or in the worst case perhaps just treat them like skipped tests?

**Screenshots**

<img width="638" alt="Screen Shot 2022-05-19 at 19 17 29" src="https://user-images.githubusercontent.com/8559/169435438-042d5be2-01e1-4c97-900c-ee58640f17f8.png">

**Environment**
 - OS: macOS
 - VSCode version: 1.67.2 (Universal)
 - Vitest version: 0.12.6
- Vitest plugin version: v0.2.3

**Comment by zxch3n** [comment]
Fixed in v0.2.10. You can switch to the pre-release version to test whether the issue persists

**Comment by beorn** [comment]
Confirms that it works - thanks! 😁
